web/yo/app/scripts/controllers/gene.js

```diff
@@ -2619,8 +2626,11 @@ angular.module('oncokbApp')
                     }
                 });
             };
+            function checkObsolete(eStatus) {
```

Comment:
This function can be changed to `checkObsolete(object, key)` and use 'name' as default value for the key.  And all `checkObsolete(mutation.name_eStatus)` can be changed to `checkObsolete(mutation)` or `checkObsolete(mutation, 'name')`

Inside the function, use `return object[key_'eStatus'] && object[key + '_eStatus'].get('obsolete') === 'true'`